### DISCUSSION TITLE
Images in text

### DISCUSSION THREAD

**Discussion by PLasmaCarrotGun** [squidfunk/mkdocs-material#5650] [Q&A]
Hello!

Is there a way to place pictures (icons) inside text like this:

Text ![image](https://github.com/squidfunk/mkdocs-material/assets/90922590/d552a58c-c09e-4f30-a17c-0ef0a0315b87) text

Thank you in advance!

**Comment by hendrikp** [comment]
Yes ofc, you kinda provided your sample yourself, you need to place the images relative to your markdown files or use absolute urls.

```
Text ![](https://user-images.githubusercontent.com/90922590/248808709-d552a58c-c09e-4f30-a17c-0ef0a0315b87.png) text
```

Do not fill the image title, to be sure it works, some plugins otherwise generate figures out it.

Alternatively you can use the emojis / icon library of mkdocs-material:
https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/
